gcloud-java-storage/src/main/java/com/google/gcloud/storage/StorageImpl.java

```diff
@@ -577,6 +577,68 @@ public URL signUrl(BlobInfo blobInfo, long expiration, SignUrlOption... options)
     }
   }
 
+  @Override
+  public List<BlobInfo> get(BlobInfo blobInfo1, BlobInfo blobInfo2, BlobInfo... blobInfos) {
```

Comment:
Wasn't `get(BlobInfo blobInfo, BlobInfo blobInfos)` not enough?

I assume that when called with get(blobInfo) the singular version would be picked vs the the vargs, no?